sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -15,33 +14,12 @@ public CertificateClientLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
-        [Test]
-        public async Task VerifyCertificateCreateDefaultPolicy()
-        {
-            string certName = Recording.GenerateId();
-
-            CertificateOperation operation = await Client.StartCreateCertificateAsync(certName);
-
-            RegisterForCleanup(certName);
-
-            CertificateWithPolicy certificate = await WaitForCompletion(operation);
-
-            Assert.NotNull(certificate);
-
-            Assert.AreEqual(certificate.Name, certName);
-        }
-
-
         [Test]
         public async Task VerifyGetCertificateOperation()
         {
             string certName = Recording.GenerateId();
 
-            CertificatePolicy certificatePolicy = Client.CreateDefaultPolicy();
-
-            certificatePolicy.IssuerName = "UNKNOWN";
-
-            await Client.StartCreateCertificateAsync(certName);
+            await Client.StartCreateCertificateAsync(certName, DefaultPolicy);
```

Comment:
The default policy is not equivalent to the lines you reference above. DefaultPolicy has issuer type "Self", whereas this test relies on the certificate issuer being "UNKNOWN" so that the signing operation will not complete. Also the three tests below have the same issue.

Reply:
Thanks for catching. Likely just overzealous copy/paste.